### ISSUE TITLE
Error at build

### ISSUE THREAD

**Issue by Susie-B** [alsa-project/snd-firewire-ctl-services#179]
Hi,

I tried to build snd-firewire-ctl-services with the command below and failed.

`	root@darkstar:/tmp/snd-firewire-ctl-services-master# cargo build 2>./log`

The command gave me a very long error log attouched below . 

[log.txt](https://github.com/alsa-project/snd-firewire-ctl-services/files/14118687/log.txt)

 I'm sorry I can't read what was wrong from the log because I have no knowledge of the cargo build system.

Of course, before building it, I had installed its dependencies, the latest version of libhinawa, libhitaki and alsa-gobject under /usr/local.

I would appreciate any advice or suggestions for the proper build.
 
Thanks,
Susie B.

**Comment by takaswie** [comment]
H @Susie-B i

Thanks for the report.

> I'm sorry I can't read what was wrong from the log because I have no knowledge of the cargo build system.

```
$ grep undefined log.txt
          /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/hinawa-0.7.1/src/fw_req.rs:125: undefined reference to `hinawa_fw_req_transaction_sync'
          /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/hinawa-0.7.1/src/fw_req.rs:125: undefined reference to `hinawa_fw_req_transaction_sync'
          /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/hinawa-0.7.1/src/fw_req.rs:125: undefined reference to `hinawa_fw_req_transaction_sync'
```

I guess libhinawa v4.0.0 (the latest) is installed in your system, while current snd-firewire-ctl-services still depend on v2.6
via hinawa crate v0.7.1, since I do not push any change to update it yet.

The intallation of libhinawa v2.6.1 would solve your issue.

https://git.kernel.org/pub/scm/libs/ieee1394/libhinawa.git/tag/?h=2.6.1

Regards

**Comment by Susie-B** [comment]
Hello, Sakamoto-san.

I have successfully installed snd-firewire-ctl-services with libhinawa of ver. 2.6.1.
I will test it on my audiofire4 later.

I appreciate your prompt assistance.

Best regards,
Susie B.

**Comment by takaswie** [comment]
> I have successfully installed snd-firewire-ctl-services with libhinawa of ver. 2.6.1.

Good.

Within the next few months, I'll put the series of changes to switch the version of libhinawa dependency, but not now.
